templates/configmap.yaml

```diff
@@ -8,6 +8,10 @@ data:
   configuration.py: |
     import re
     import yaml
+    import ldap
+
+    from django_auth_ldap.config import LDAPSearch
+    from importlib import import_module
```

Comment:
I don't think this change is needed, is it? Does this not break with the non-LDAP images?

Reply:
You're correct - this should not be there and did not occur to us, as we've used LDAP images
Without this there should be no change for non-ldap images